R-package/R/lgb.plot.importance.R

```diff
@@ -1,68 +1,68 @@
 #' Plot feature importance as a bar graph
-#' 
```

Comment:
as I've mentioned before, I have my `RStudio` (the IDE I use to edit R code) setup to automatically trim horizontal whitespace whenever I save a `.R` file. If these lines are annoying and you want me to undo them please do let me know and I'd be happy to.

Reply:
Maybe you can create a separate PR for this purpose for other files which haven't been edited by you?

I mean, just open every file in the package and send a PR? 😃 

Don't you mind, @Laurae2 ?

Reply:
@jameslamb Either you do it or I'll do it in a new PR.

I have also this setting trailing whitespace removal setting on my productive 📦 (but not on development 📦 ), so I always see how funny it gets in git to see all those whitespaces removal when moving from development 📦 to production 📦

Reply:
ha ok! I'll do it in a new PR. I think that type of thing really belongs in its own change.

Reply:
created #1642 so I don't forget